packages/x-date-pickers-pro/package.json

```diff
@@ -82,7 +82,7 @@
     "@types/luxon": "^3.1.0",
     "date-fns": "^2.29.3",
     "dayjs": "^1.11.7",
-    "luxon": "^1.28.0 || ^2.0.0 || ^3.0.2",
```

Comment:
It's the `devDeps` so like on the community package I'm just putting the last version here